ui/src/components/events/getWeek.js

```diff
@@ -0,0 +1,15 @@
+export default function isDateInThisWeek(date) {
+  const todayObj = new Date();
+  const eventDate = new Date(new Date(date).toDateString());
+  const todayDate = todayObj.getDate();
+  const todayDay = todayObj.getDay();
+
+  const firstDayOfWeek = new Date(
+    new Date(todayObj.setDate(todayDate - todayDay)).toDateString()
+  );
```

Comment:
I think doing it like this causes sunday to be the first day of the week so sunday celebration wont show up in this filter. maybe can set Monday as start of week instead